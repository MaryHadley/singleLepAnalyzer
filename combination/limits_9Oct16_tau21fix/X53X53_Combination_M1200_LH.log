000 (1295365.000.000) 10/09 22:41:04 Job submitted from host: <192.168.63.249:41533>
...
001 (1295365.000.000) 10/09 22:41:26 Job executing on host: <192.168.63.235:42190>
...
006 (1295365.000.000) 10/09 22:41:34 Image size of job updated: 175440
	75  -  MemoryUsage of job (MB)
	75920  -  ResidentSetSize of job (KB)
...
006 (1295365.000.000) 10/09 22:46:34 Image size of job updated: 283404
	163  -  MemoryUsage of job (MB)
	165980  -  ResidentSetSize of job (KB)
...
006 (1295365.000.000) 10/10 18:33:37 Image size of job updated: 297936
	163  -  MemoryUsage of job (MB)
	165980  -  ResidentSetSize of job (KB)
...
005 (1295365.000.000) 10/10 18:42:25 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 19:59:19, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 19:59:19, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	1594  -  Run Bytes Sent By Job
	307  -  Run Bytes Received By Job
	1594  -  Total Bytes Sent By Job
	307  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :    31721        1   3824810
	   Memory (MB)          :      163     3072      3072
...
