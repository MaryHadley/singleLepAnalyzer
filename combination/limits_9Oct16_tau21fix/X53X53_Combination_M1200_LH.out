/var/lib/condor/execute/dir_1927
Total number of likelihood evaluations: 1100000101
Total number of likelihood evaluations: 11000101
[INFO]  executing script /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M1200_LH.py
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M1200_LH/quant-toys:0-sig-9ecb400645.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M1200_LH/quant-data-sig-4c10c76d59.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  workdir is /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M1200_LH
