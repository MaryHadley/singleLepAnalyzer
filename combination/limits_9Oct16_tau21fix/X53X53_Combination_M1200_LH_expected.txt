# x; y; band 0 low; band 0 high; band 1 low; band 1 high
         0   0.079965   0.040526    0.16608  0.055388    0.11733
